dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -11,7 +11,7 @@
 
 from dataprofiler.profilers import CategoricalColumn
 from dataprofiler.profilers.profile_builder import StructuredColProfiler
-
+from ...profilers.profiler_options import CategoricalOptions
```

Comment:
Don't do relative imports in test modules, use absolute here. Relative imports need to be used in the code itself though, but tests should exist separate of the library, not as if they live within the library, if that makes sense.